specification/machinelearningservices/AzureAI.Evaluation/model.tsp

```diff
@@ -0,0 +1,139 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./common.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Microsoft.MachineLearningServices;
+
+@doc("Evaluator Configuration")
+model EvaluatorConfiguration {
+  @doc("Identifier of the evaluator.")
+  id: string;
+
+  @doc("Initialization parameters of the evaluator.")
+  initParams?: Record<string>;
+
+  @doc("Data parameters of the evaluator.")
+  dataMapping?: Record<string>;
+}
+
+@doc("Abstract data class.")
+@discriminator("type")
+model Data {
+  @doc("Data identifier.")
+  id: string;
+}
+
+@doc("Data Source for Application Insight.")
+model AppInsight extends Data {
+
+  type: "app_insights";
+
+  @doc("Application Insight connection string.")
+  connectionString: string;
+
+  @doc("Query to fetch data.")
+  query: string;
+}
+
+@doc("Dataset as source for evaluation.")
+model Dataset extends Data {
+  type: "dataset";
+}
+
+@discriminator("type")
+@doc("Evaluation Target.")
+model EvaluationTarget {
+  @doc("Name of the evaluation target.")
+  name?: string;
+}
+
+@doc("Base Model and prompt for evaluation.")
+model BaseModel extends EvaluationTarget {
+  @doc("Base model for evaluation.")
+  endpoint: string;
+
+  @doc("System prompt to be used with base model.")
+  prompt: string;
+
+  @doc("parameters for evaluation.")
+  parameters: Record<string>;
+
+  type: "base_model";
+}
+
+@doc("Evaluation Definition")
+model Evaluation {
+  @doc("Identifier of the evaluation.")
+  id?: string;
+
+  @doc("Data for evaluation.")
+  data: Data;
+
+  @doc("Update stage to 'Archive' to archive the asset. Default is Development, which means the asset is under development.")
+  displayName?: string;
+
+  @doc("Description of the evaluation. It can be used to store additional information about the evaluation and is mutable.")
+  description?: string;
+
+  @doc("Metadata containing createdBy and modifiedBy information.")
+  @visibility("read")
+  systemData?: SystemData;
+
+  @doc("Status of the evaluation. It is set by service and is read-only.")
+  @visibility("read")
+  status?: string;
+
+  @doc("Evaluation's tags. Unlike properties, tags are fully mutable.")
+  tags?: Record<string>;
+
+  @doc("Evaluation's properties. Unlike tags, properties are add-only. Once added, a property cannot be removed.")
+  properties?: Record<string>;
+
+  @doc("Evaluators to be used for the evaluation.")
+  evaluators: Record<EvaluatorConfiguration>;
+
+  @doc("Output location for the evaluation.")
+  outputLocation?: string;
+
+  @doc("Evaluation Target.")
+  evaluationTarget?: EvaluationTarget;

```

Comment:
Ethan and me were discussing, if we wanted to keep this one out of the contract for Ignite since we are not supporting it. Thoughts?
We can add validation in service but worried we will have customer tickets for this field.